junit4-runner/src/main/java/org/junit/gen5/junit4/runner/JUnit5.java

```diff
@@ -131,38 +130,29 @@ private void addFiltersFromAnnotations(DiscoveryRequest spec) {
 	}
 
 	private List<DiscoverySelector> getPackageSpecificationElements() {
-		return stream(getPackageNames()).map(PackageTestPlanSpecificationElementBuilder::byPackageName).collect(
-			toList());
+		return stream(getPackageNames()).map(PackageSelectorBuilder::byPackageName).collect(toList());
```

Comment:
True. I didn't introduce this intentionally. Especially, as we wanted to remove the builder in the next step. I will change this line.